teuthology/misc.py

```diff
@@ -654,6 +654,7 @@ def delete_file(remote, path, sudo=False, force=False):
     remote.run(
         args=args,
         stdout=StringIO(),
+        check_status = check
```

Comment:
nitpick, but please drop the spaces around the `=`

Reply:
I can add cosmetic fix as separate commit.